### PR TITLE
Add basic kyber assertions

### PR DESCRIPTION
**Pull request by asoong**

### CODE REVIEW

File: test/helpers/exchangeHelpers.ts [hunk 1]
```diff
@@ -93,3 +95,45 @@ export const deployZeroExExchangeWrapperContract = async (
...
+  kyberNetworkProxyAddress: Address,
+  transferProxy: TransferProxyContract,
+  core: CoreContract,
+  provider: Provider,
+): Promise<KyberNetworkWrapperContract> => {
+  const web3 = new Web3(provider);
+
+  const truffleKyberNetworkWrapperContract = contract(KyberNetworkWrapper);
+  truffleKyberNetworkWrapperContract.setProvider(provider);
+  truffleKyberNetworkWrapperContract.setNetwork(50);
```
Comment (felix2feng) [inline test/helpers/exchangeHelpers.ts]: Why set network to 50? Is that our chain default?

File: test/helpers/index.ts [hunk 2]
```diff
@@ -12,6 +12,13 @@ export {
...
   addAuthorizationAsync,
   getTokenBalances,
+  tokenDeployedOnSnapshot,
 } from './coreHelpers';
-export { deployTakerWalletWrapperContract, deployZeroExExchangeWrapperContract } from './exchangeHelpers';
+
+export {
+  deployKyberNetworkWrapperContract,
+  deployTakerWalletWrapperContract,
+  deployZeroExExchangeWrapperContract
```
Comment (justinkchen) [inline test/helpers/index.ts]: nit trailing comma

File: test/helpers/exchangeHelpers.ts (hunk 1 above)
Comment (asoong) [reply test/helpers/exchangeHelpers.ts]: Yes

File: src/assertions/OrderAssertions.ts [hunk 3]
```diff
@@ -345,17 +326,61 @@ export class OrderAssertions {
...
+    // Kyber trade source token is equivalent to the issuance order maker token
+    this.commonAssertions.isEqualString(
+      issuanceOrder.makerToken,
+      trade.sourceToken,
+      orderErrors.MAKER_TOKEN_AND_KYBER_SOURCE_TOKEN_MISMATCH(),
+    );
+
+    // Kyber destination token is a component of the Set
+    await this.setTokenAssertions.isComponent(issuanceOrder.setAddress, trade.destinationToken);
+
```
[the remark this answers is not in the document]
Comment (asoong) [reply src/assertions/OrderAssertions.ts]: +1

File: test/helpers/index.ts (hunk 2 above)
Comment (asoong) [reply test/helpers/index.ts]: +1

**Pull request merged**